.github/workflows/builds.yml

```diff
@@ -3,65 +3,85 @@ name: Build bpftools archives
 on: [push, pull_request]
 
 jobs:
-  build_bpftools:
-    name: Build bpftools archive
+  build_jdwp_project:
+    name: Build and Test JDWP Project
     runs-on: ubuntu-22.04
-    strategy:
-      matrix:
-        env:
-        - ARCH: arm64
-        - ARCH: x86_64
+    
     steps:
     - uses: actions/checkout@v3
-    - name: Install deps
-      run: sudo ./scripts/jammy-install-deps.sh
-    - name: Setup ndk
-      run: ./scripts/download-ndk.sh
-    - name: Build
-      env: ${{matrix.env}}
-      run: make bpftools NDK_PATH=`pwd`/android-ndk-r23b NDK_ARCH=${{ matrix.env['ARCH'] }}
-    - uses: actions/upload-artifact@v3
-      with:
-        name: bpftools-android-${{ matrix.env['ARCH'] }}.tar.gz
-        path: bpftools-${{ matrix.env['ARCH'] }}.tar.gz
-  build_bpftools_min:
-    name: Build bpftools-min archive
-    runs-on: ubuntu-22.04
-    strategy:
-      matrix:
-        env:
-        - ARCH: arm64
-        - ARCH: x86_64
-    steps:
-    - uses: actions/checkout@v3
-    - name: Install deps
-      run: sudo ./scripts/jammy-install-deps.sh
-    - name: Setup ndk
-      run: ./scripts/download-ndk.sh
-    - name: Build
-      run: make bpftools-min NDK_PATH=`pwd`/android-ndk-r23b NDK_ARCH=${{ matrix.env['ARCH'] }}
-    - uses: actions/upload-artifact@v3
-      with:
-        name: bpftools-min-android-${{ matrix.env['ARCH'] }}.tar.gz
-        path: bpftools-min-${{ matrix.env['ARCH'] }}.tar.gz
-  build_bpftrace_static:
-    name: Build static bpftrace binaries
-    runs-on: ubuntu-22.04
-    strategy:
-      matrix:
-        env:
-        - ARCH: arm64
-        - ARCH: x86_64
-        - ARCH: armv7
-    steps:
-    - uses: actions/checkout@v3
-    - name: Install deps
-      run: sudo ./scripts/jammy-install-deps.sh
-    - name: Setup ndk
-      run: ./scripts/download-ndk.sh
-    - name: Build
-      run: make bpftrace NDK_PATH=`pwd`/android-ndk-r23b NDK_ARCH=${{ matrix.env['ARCH'] }} STATIC_LINKING=true LLVM_BPF_ONLY=true
-    - uses: actions/upload-artifact@v3
-      with:
-        name: bpftrace-android-${{ matrix.env['ARCH'] }}
-        path: out/android/${{ matrix.env['arch'] }}/bin/bpftrace
+    - name: Navigate to Parent Directory
+      run: cd ..  
+    - name: Initialize submodule folder
+      run: git submodule init && git submodule update
+    - name: Initialize JDWP Project Dependencies
+      run: make buck2-host python-host pyre-host
+    - name: Setup path environment
+      run: eval `make setup-env`
+    - name: Run Buck2 Build
+      run: out/host/bin/buck2 run //projects/jdwp:main
+    - name: Run Buck2 Tests
+      run: out/host/bin/buck2 test //projects/jdwp/tests/...
+    - name: Run Pyre Type Checking
+      run: cd projects/jdwp && out/host/bin/pyre
```

Comment:
This might not work because after entreing `projects/jdwp` `out/bin/pyre` refers to `ExtendedAndroidTools/projects/jdwp/out/bin/pyre` instead of `ExtendedAndroidTools/out/bin/pyre`

Reply:
Yeah. Didn't see that. My bad.